continuum/task_set.py

```diff
@@ -70,15 +73,36 @@ def plot(
 
     def __len__(self) -> int:
         """The amount of images in the current task."""
-        return self.x.shape[0]
+        return self._x.shape[0]
+
+    def get_samples_from_ind(self, indices):
```

Comment:
We should agree whether we use `indices` or `indexes`. I've used the latter everywhere.

Reply:
ok